tests/acceptance/features/apiAuthApp/token.feature

```diff
@@ -157,4 +157,37 @@ Feature: create auth-app token
     And the content in the response should include the following content:
       """
       error parsing expiry. Use e.g. 30m or 72h
-      """
\ No newline at end of file
+      """
+
+  @env-config
+  Scenario: admin creates auth-app token with user-id
+    Given the config "AUTH_APP_ENABLE_IMPERSONATION" has been set to "true"
+    When user "Admin" creates app token with user-id for user "Alice" with expiration time "72h" using the auth-app API
+    Then the HTTP status code should be "200"
+    And the JSON data of the response should match
+      """
+      {
+        "type": "object",
+        "required": ["token","expiration_date","created_date","label"],
+        "properties": {
+          "token": { "pattern": "^[a-zA-Z0-9]{16}$" },
+          "label": { "const": "Generated via API" }
+        }
+      }
+      """
+
+
+  Scenario: user creates auth-app token with user-id
+    When user "Alice" creates app token with user-id for user "Alice" with expiration time "72h" using the auth-app API
```

Comment:
What happens when the user tries to create a token with the next user ` user-id ` ? 
Could you increase coverage?

Reply:
you mean not self but another user

Reply:
yeah